### ISSUE TITLE
Global variable "performance" is being overwritten

### ISSUE THREAD

**Issue by amstech** [fraywing/textAngular#1401]
There's a declaration of global variable "performance" on line 53 of v.1.5.6.0 that is overwriting global variable" performance" which is provided by the browser.  Original variable is an object which contains properties like "timing".  This property is being used by other application/services (in our case TokBox) which are relying on the object to be in its original form.  Renaming this variable to a different name is fixing the issue.  This is an issue in IE11, other browsers do not seems to have this problem.

Original Code: 

```
var performance = performance || {};
performance.now = (function () {
    return performance.now ||
		performance.mozNow ||
		performance.msNow ||
		performance.oNow ||
		performance.webkitNow ||
		function() { return new Date().getTime(); };
})();
```

**Comment by JoelParke** [comment]
YES - this code is there to provide access to exactly the same things.  What does IE11 provide?  Under Chrome there is no performance and under Firefox there is no performance as you note.

**Comment by amstech** [comment]
Here's the [link](https://msdn.microsoft.com/en-us/library/ff974724(v=vs.85).aspx).

As you can see, they're expecting "timing" off of performance object.

**Comment by JoelParke** [comment]
I just tested with under Edge and Internet Explorer 11 for textAngular 1.5.13 and nothing is being over written in either case.  So I am unsure what the issue is.

**Comment by amstech** [comment]
As I mentioned before, we are using textAngular with another service TokBox.  Here's the url to their file: http://static.opentok.com/v2/js/opentok.js.  Please search for "performance.timing.navigationStart".  This line is throwing an exception "Unable to get property 'navigationStart' of undefined or null reference".  While debugging, we noticed that performance is overwritten and no longer has object "timing".

**Comment by JoelParke** [comment]
<p>I will try and take a look at this, but it seems that they are doing something a bit non-standard.  But perhaps I can rename this to keep the conflict from appearing in your special case.  For Example: performance -&gt; ta-performance.  </p>
